### ISSUE TITLE
How can I specify a particular instance for stream_circuit_logger.py?

### ISSUE THREAD

**Issue by annymous** [meejah/txtorcon#202]
I am using tor-instance-create. It appears that stream_circuit_logger.py is only reporting circuits and streams for the main instance. How can I specify a particular instance to log?

Also, can I log multiple instances? And will "[TorControlProtocol,client]" indicate the instance for each output line?

**Comment by meejah** [comment]
It's reporting events for the one Tor instance it's connected to. You can connect to multiple instances, yes -- simply make another connection to another control-port.

Questions like this are probably better asked on `#tor-dev` on OFTC or via the tor-dev mailing-list.

**Comment by annymous** [comment]
Thank you. I have studied the documentation, and perused relevant scripts, and I understand the need to connect to other control ports. I see that one must specify "endpoint = unix:/var/run/tor-instances/linkN/control", but I do not understand how to do that. I have tried adding it to torcontrolprotocol.py and stream_circuit_logger.py and experimented with using it as a command option. But nothing worked.

If you would be so kind as to explain how to do this, I would be happy to draft an insertion for the documentation.
